tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -58,6 +58,14 @@ var _ = Describe("odo devfile create command tests", func() {
 		})
 	})
 
+	Context("When executing odo create with incorrect devfile component", func() {
```

Comment:
@amitkrout Does `incorrect` devfile and `invalid` differs? I think we do have spec for the same https://github.com/openshift/odo/pull/3314/files#diff-5291a6a42f2148d4c3e48ea4b73cff23R136 . Correct me if I am missing something.

Reply:
The original pr https://github.com/openshift/odo/pull/3265 its clearly mentioned in the pr description that the component mane is invalid not incorrect. so I replaced invalid with incorrect to be in more sync with the original pr. 

@prietyc123 Does it make sense now ?

Reply:
Yes, that's true we have `invalid` component name but what I mean is the spec that you are adding is duplicate of line no 136 https://github.com/openshift/odo/pull/3314/files#diff-5291a6a42f2148d4c3e48ea4b73cff23R136 .

Reply:
Ahhh...Got it. You know its the Monday effect hahaha...

Reply:
Good catch !!!